<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>235</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tic Tac Toe</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>219</width>
      <height>185</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="TitleLabel">
       <property name="font">
        <font>
         <family>Felix Titling</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Tic Tac Toe</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="MaxGameLabel">
         <property name="font">
          <font>
           <family>Felix Titling</family>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Max Score (1-10)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ScoreLineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="FPlayerNameLabel">
         <property name="font">
          <font>
           <family>Felix Titling</family>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>First Player Name (Circle)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="FPlayerNameEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="SPlayerNameLabel">
         <property name="font">
          <font>
           <family>Felix Titling</family>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Second Player Name (Cross)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="SPlayerNameEdit"/>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <family>Felix Titling</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
